pkg/distro/defs/fedora/distro.yaml

```diff
@@ -1796,3 +1870,121 @@ image_types:
+
+  # Based on lorax runtime-install.tmpl
+  "everything-netinst":
+    name_aliases: ["netinst"]
+    filename: "netinst.iso"
+    mime_type: "application/x-iso9660-image"
+    bootable: true
+    boot_iso: true
+    image_func: "netinst"
+    iso_label: "Everything"
+    build_pipelines: ["build"]
+    payload_pipelines:
+      - "anaconda-tree"
+      - "efiboot-tree"
+      - "bootiso-tree"
+      - "bootiso"
+    exports: ["bootiso"]
+    required_partition_sizes: *default_required_dir_sizes
+    installer_config: *default_installer_config
+    image_config:
+      locale: "en_US.UTF-8"
+      iso_rootfs_type: "squashfs"
+      condition:
+        version_less_than:
+          41:
+            iso_rootfs_type: "squashfs-ext4"
+
+    package_sets:
+      installer:
+        - include:
+            - anaconda
+            - anaconda-widgets
+            - anaconda-install-img-deps
+            - ostree
+            - rpm-ostree
+            - pigz
+            - audit
+            - bind-utils
+            - kernel
+            - kernel-modules
+            - kernel-modules-extra
+            - grubby
+            - glibc-all-langpacks
+            - plymouth
+            - anaconda-dracut
+            - dracut-network
+            - dracut-config-generic
+            - initscripts
+            - cryptsetup
+            - rpcbind
+            - kbd
+            - kbd-misc
+            - tar
+            - xz
+            - curl
+            - bzip2
+            - rsyslog
+            - device-mapper-persistent-data
+            - xfsdump
+            - volume_key
+            - nss-tools
+            - selinux-policy-targeted
+            - audit
+            - ethtool
+            - openssh-server
+            - nfs-utils
+            - openssh-clients
+            - tigervnc-server-minimal
+            - tigervnc-server-module
+            - net-tools
+            - bridge-utils
+            - nmap-ncat
+            - prefixdevname
+            - pciutils
+            - usbutils
+            - ipmitool
+            - mt-st
+            - smartmontools
+            - hdparm
+            - rdma-core
+            - rng-tools
+            - nvme-cli
+            - default-fonts-core-sans
+            - default-fonts-other-sans
+            - google-noto-sans-cjk-fonts
+            - gdb-gdbserver
+            - libreport-plugin-bugzilla
+            - libreport-plugin-reportuploader
+            - fpaste
+            - python3-pyatspi
+            - nano
+            - nano-default-editor
+            - vim-minimal
+            - strace
+            - lsof
+            - dump
+            - less
+            - wget2-wget
+            - rsync
+            - bind-utils
+            - ftp
+            - mtr
+            - vconfig
+            - spice-vdagent
+            - gdisk
+            - hexedit
+            - sg3_utils
+          condition:
+            architecture:
+              x86_64:
+                include:
```

Comment:
(maybe a silly question) given that this image-type only uses platforms "x86_64" and "aarch64", do we need this conditional? Or should we just add `dmidecode` to the "default" package-set? I guess if the plan is that netinst becomes available on more arches it makes sense to already have the condition here.